README.md

```diff
@@ -1,27 +1,26 @@
-# mlx_steering_dump
-This is Mellanox SW steering Dump Tool.
+# mlx_steering_dump_parser
+This is Mellanox SW steering Dump file Parser.
 
 How to trigger a dump.file generation
 =====================================
 Dump generation is done using rdma-core,
 it can be done directly by calling mlx5dv_dump_dr_domain API
 or using mlx5_pmd
 
-dump example
+dump file example
 ============
-example.json
+example.csv
 
 Running examples
 ================
 To show steering rules run:<br/>
-	mlx_steering_dump.py -f `pwd`/example.json<br/>
+<pre>mlx_steering_dump_parser.py -f `pwd`/example.csv</pre>
 To show steering tree/hierarchy run:<br/>
-	mlx_steering_dump.py -f `pwd`/example.json -t<br/>
-To trigger DPDK app run for pid=5336 and mlx5_port=0:<br/>
-	mlx_steering_dump/mlx_steering_dump.py -d 5336 0 -f /tmp/my_json_dump<br/>
-To trigger testpmd app:<br/>
-	mlx_steering_dump.py -d `pgrep testpmd` 0  -f /tmp/my_json_dump -t<br/>
+<pre>mlx_steering_dump_parser.py -f `pwd`/example.csv -t</pre>
+To trigger DPDK app(PID=5336, port=0) run:<br/>
```

Comment:
1. Need space before "(PID"
2. Why do you need the port number parameter? Why to give the user this option?
3. It's better not to use numbers here. For instance, (assuming you loose the port number param) the example can be as follows:

mlx_steering_dump_parser.py -d <DPDK PID> -f  <dump_file> </pre>

4. It's better to use "-p" (for PID) and not "-d"